### PR TITLE
Overall copy updates

### PR DESCRIPTION
**Pull request by JaneenBredenkamp**
There was a request to bring forward the Linkedin profile and Blog, for which I understand why. If there are issues related to this or any other copy updates please talk to Jon.

### CODE REVIEW

File: anyone.html [hunk 1]
```diff
@@ -84,7 +84,7 @@
 					<div class="page copy animation-element fade-up">
 						<p class="page-heading u-white"> Sporty People </p>
 						<p class="page-text">
-							Find someone from the office to have a friendly squash game with after work. Or a neighbour down the road to join you for your morning runs. Or get an amateur football league together. Kalido can find you people nearby who share your love of the beautiful game (or any game).
+							Find someone from the office to have a friendly squash game with after work. Or a neighbour down the road to join you for your early morning runs. Kalido can find people nearby who share your passion. Just tell Kalido you want to meet &lsquo;a running partner&rsquo;; or a &lsquo;gym buddy&rsquo; or a &lsquo;squash partner&rsquo;. You name the sport, we&#700;ll find the person.
```
Comment (Leandri-Viviers) [inline anyone.html]: remove double semi-colon

File: anyone.html [hunk 2]
```diff
@@ -112,7 +112,7 @@
 					<div class="page copy animation-element fade-up">
 						<p class="page-heading u-white"> School People </p>
 						<p class="page-text">
-							Wonder what your old school friends are up to? With Kalido&#700;s handy notifications, you&#700;ll know whenever one of them is in your area, so you can nip out quickly and catch up, instead of scheduling something 3 months from now. And of course, they can return the favour by entertaining you when you&#700;re in <i>their</i> area.
+							Wonder what your old school friends are up to? With Kalido&#700;s handy notifications, you&#700;ll know whenever one of them is in your area, so you can nip out quickly and catch up, instead of scheduling something 3 months from now. And of course, they can return the favour by entertaining you when you&#700;re in <i>their</i> area. Just tell Kalido you want to meet &lsquo;people who went to [your school]&rsquo; or &lsquo;people who worked at [your company]&rsquo;.
```
Comment (Leandri-Viviers) [inline anyone.html]: what is this [your school] [your company]? Just remove the brackets, it doesn't make sense

File: app/copyright_distribution/index.html [hunk 3]
```diff
@@ -93,10 +93,10 @@
...
-				Details of Designated agent: <a href="mailto:info@kalido.me">info@kalido.me</a>	
+				Details of Designated Agent: <a href="mailto:info@kalido.me">info@kalido.me</a>
 			</p>
 		</div>
 	</body>
 </html>
 
-<!-- Small:14pt, Medium:16pt -->
\ No newline at end of file
+<!-- Small:14pt, Medium:16pt -->
```
Comment (Leandri-Viviers) [inline app/copyright_distribution/index.html]: You can remove these comments, don't think they're necessary any more

File: app/terms_of_use/index.html [hunk 4]
```diff
@@ -156,17 +156,17 @@
 			<p class="text">
 				Should a dispute arise about these Terms, or about something related to these Terms, the dispute will be addressed in San Francisco County, California, in English, in accordance with the Streamlined Arbitration Rules and Procedures of Judicial Arbitration and Mediation Services, Inc. (&lsquo;JAMS&rsquo;).<br/><br/>
 
-				To keep things clear, use of Kalido, and the acceptance of these Terms, constitutes consent to the exclusive jurisdiction and venue in the state or federal courts located in, respectively, San Francisco County, California, or the Northern District of California.<br/><br/>
+				To keep things clear, use of Kalido, and the acceptance of these Terms, constitutes consent to the exclusive jurisdiction and venue in the state or federal courts located in, respectively, San Francisco County, California, or the Northern District of California.
```
Comment (Leandri-Viviers) [inline app/terms_of_use/index.html]: Can't tell from context, why did you remove the breaks here?

File: app/terms_of_use/index.html [hunk 5]
```diff
@@ -156,17 +156,17 @@
...
 			</p>
 
-			<p class="heading"> Enforceability of terms </p>
+			<p class="heading"> Enforceability of Terms </p>
 
 			<p class="text">
 					Should any part or parts of these Terms be unenforceable or invalid, only that portion shall be removed or deleted. Removal or deletion of certain portions does not affect any rights or obligation under the rest of these Terms.<br/><br/>
 
-					These Terms, our Privacy Policy, Community Guidelines, and in app permissions form the complete contract between you and us, and no other written or oral agreements shall supersede these.<br/><br/>
+					These Terms, our Privacy Policy, Community Guidelines, and in&#8208;app permissions form the complete contract between you and us, and no other written or oral agreements shall supersede these.<br/><br/>
```
Comment (Leandri-Viviers) [inline app/terms_of_use/index.html]: What unicode is this?

File: contact.html [hunk 6]
```diff
@@ -105,17 +105,17 @@
 							--><a class="social-link" id="twitter-circle" href="https://twitter.com/getkalido" target="_blank">
 								<img alt="Follow us on Twitter" src="img/social/twitter-circle.svg"/>
 							</a><!--
-							--><a class="social-link" id="instagram-circle" href="https://www.instagram.com/getkalido/" target="_blank">
-								<img alt="Follow us on Instagram" src="img/social/instagram-circle.svg"/>
+							--><a class="social-link" id="linkedin-circle" href="https://www.linkedin.com/company-beta/18064339" target="_blank">
```
Comment (Leandri-Viviers) [inline contact.html]: :+1:

File: professionals.html [hunk 7]
```diff
@@ -84,7 +84,7 @@
 					<div class="page copy animation-element fade-up">
 						<p class="page-heading u-white"> Smart Matches </p>
 						<p class="page-text">
-							Kalido intelligently matches you to professionals who can provide the service you need. You don&#700;t even have to know their proper job title. So, if you want someone to &lsquo;fix my house&rsquo;, we&#700;ll find you the best &lsquo;building contractor&rsquo; out there. Kalido will also rank professionals based on location, your shared contacts, networks, and affiliations. This makes it easy for you to choose the best person for the job.
+							Kalido intelligently matches you with the professionals you need. You don&#700;t even have to know their proper job title. So, if you want someone to ‘fix my house&#700;, we&#700;ll find you the best ‘building contractor&#700; out there. Kalido will also rank professionals based on location and social proximity (whether they have shared contacts, networks, or affiliations with you). This makes it easy for you to choose the best person for the job.
```
Comment (Leandri-Viviers) [inline professionals.html]: Fix unicodes please

File: app/terms_of_use/index.html (hunk 5 above)
Comment (JaneenBredenkamp) [inline app/terms_of_use/index.html]: It's the unicode for "-", will change it to "-"

File: app/terms_of_use/index.html (hunk 4 above)
Comment (JaneenBredenkamp) [inline app/terms_of_use/index.html]: the double breaks were right before a </p>, which creates more space. It looked weird as well because it's the only place with such a big space
